tests/scripts/generate_test_keys.py

```diff
@@ -7,16 +7,12 @@
 generating the required key at run time. This helps speeding up testing."""
 
 import os
-import sys
 from typing import Iterator
 import re
-# pylint: disable=wrong-import-position
-SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__)) + "/"
-sys.path.append(SCRIPT_DIR + "../../scripts/")
-from mbedtls_dev.asymmetric_key_data import ASYMMETRIC_KEY_DATA
 import scripts_path # pylint: disable=unused-import
+from mbedtls_dev.asymmetric_key_data import ASYMMETRIC_KEY_DATA
 
-OUTPUT_HEADER_FILE = SCRIPT_DIR + "../src/test_keys.h"
+OUTPUT_HEADER_FILE = os.path.dirname(os.path.abspath(__file__)) + "/../src/test_keys.h"
```

Comment:
Sorry for not catching this earlier, and not a blocker because we aren't doing this consistently yet: to find the source tree, you should use the `build_tree` module.

Reply:
Can probably be improved in #9010 (unless there is other rework here).